sqlglot/tokens.py

```diff
@@ -113,6 +114,7 @@ class TokenType(AutoName):
     CURRENT_ROW = auto()
     CURRENT_TIME = auto()
     CURRENT_TIMESTAMP = auto()
+    DATE_PART = auto()
```

Comment:
this seems unnecessary, i don't think you need this, nor do you need to put it into func_tokens